converter/converting.py

```diff
@@ -403,8 +411,11 @@ def datamodel2json_conversion(submission, working_dir, logger):
         "protocol": [generate_usi_protocol_object(p) for p in submission.protocol],
         "sample": [generate_usi_sample_object(s) for s in submission.sample],
         "assay": [generate_usi_assay_object(a, submission.info) for a in submission.assay],
-        "assay_data": [generate_usi_data_object(ad, submission.info) for ad in submission.assay_data]
+        "assay_data": [generate_usi_data_object(ad, submission.info) for ad in submission.assay_data],
```

Comment:
guessing you don't actually want that last comma since there are no more hash elements?

Reply:
Good catch! Apparently Python doesn't mind the extra comma ;)